oops/oops.go

```diff
@@ -180,6 +182,11 @@ func (e *oopsError) writeStackTrace(w io.Writer) {
 				fmt.Fprintf(w, "%s\n", frame.Function)
 			}
 			fmt.Fprintf(w, "\t%s:%d\n", frame.File, frame.Line)
+			lineCount++
+            if lineCount > maxStackLength {
```

Comment:
the indentation here is weird, can we fix that?

Reply:
done